backend-project/small_eod/letters/serializers.py

```diff
@@ -13,7 +13,7 @@
 class DocumentTypeSerializer(serializers.ModelSerializer):
     class Meta:
         model = DocumentType
-        fields = ["name"]
+        fields = ["id", "name"]
```

Comment:
W innym PR widziałem, że bodajże id i name w podobnym serializerze było w dwóch liniach. wiem, że nie ma to większego znaczenia, ale może warto to zunifikować

Reply:
tym @kuskoman rządzi [Black](https://github.com/psf/black) i on na podstawie długości listy ustala, czy jedna linia czy jednak chce rozdzielać.